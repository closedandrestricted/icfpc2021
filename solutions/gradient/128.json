{"vertices":[[60,73],[58,73],[44,102],[49,176],[33,68],[27,198],[21,150],[117,103],[88,81],[78,29],[41,71],[24,115],[22,42],[50,140],[86,8],[48,172],[70,55],[84,119],[108,165],[103,215],[92,189],[85,230],[90,36],[136,169],[55,137],[81,152],[80,86],[70,147],[120,191],[94,209],[164,190],[101,62],[110,217],[94,170],[91,122],[142,178],[98,181],[184,210],[72,58],[114,145],[93,119],[147,180],[171,160],[121,192],[120,162],[120,162],[192,184],[144,147],[166,120],[112,145],[100,142],[129,118],[160,123],[136,114],[132,139],[120,167],[149,85],[152,83],[160,53],[170,109],[182,78],[181,76]]}