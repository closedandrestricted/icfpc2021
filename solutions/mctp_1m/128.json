{"vertices":[[21,148],[39,121],[9,117],[22,144],[11,123],[41,117],[19,140],[31,118],[69,124],[16,133],[20,146],[48,119],[39,117],[7,113],[7,113],[20,142],[39,120],[44,152],[42,162],[24,141],[42,163],[14,120],[18,139],[23,142],[13,127],[40,119],[69,132],[47,126],[14,168],[36,120],[19,175],[39,120],[27,193],[24,144],[22,147],[23,144],[23,145],[4,199],[46,149],[8,117],[40,115],[27,193],[51,174],[7,212],[19,117],[19,117],[21,178],[20,145],[53,141],[70,106],[44,129],[40,120],[9,117],[28,177],[22,152],[12,122],[60,179],[30,213],[51,148],[46,149],[39,182],[53,117]]}