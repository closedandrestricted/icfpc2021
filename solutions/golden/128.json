{"vertices":[[57,66],[52,67],[44,98],[47,176],[31,64],[24,198],[20,147],[47,104],[79,83],[76,29],[37,68],[23,112],[20,38],[48,140],[86,9],[45,172],[67,55],[81,119],[115,163],[106,217],[91,193],[87,230],[88,37],[141,172],[54,134],[78,152],[75,86],[67,147],[120,191],[96,209],[167,193],[98,63],[113,218],[94,171],[87,121],[97,178],[98,182],[189,211],[69,58],[111,144],[91,117],[146,178],[173,163],[121,192],[120,163],[79,160],[196,185],[141,144],[166,119],[118,136],[101,143],[140,113],[115,129],[138,112],[133,136],[124,165],[151,83],[153,80],[163,53],[171,108],[183,75],[183,76]]}